packages/monaco-editor-wrapper/src/editorAppBase.ts

```diff
@@ -43,9 +45,9 @@ export abstract class EditorAppBase {
         this.id = id;
     }
 
-    protected buildConfig(userConfig: UserConfig) {
+    protected buildConfig(userConfig: UserConfig): EditorAppBaseConfig {
         const userAppConfig = userConfig.wrapperConfig.editorAppConfig;
-        return {
+        const config = {
```

Comment:
Can restore as a direct return if we don't need the local config value, maybe was there for debugging?

Reply:
Will do.